packages/features/portfolio/src/hooks/use-portfolio-snote-table.tsx

```diff
@@ -0,0 +1,187 @@
+  DataTableColumn,
+  MultiValueCell,
+  DisplayCell,
+} from '@notional-finance/mui';
+import { TotalEarningsTooltip } from '../components';
+import {
+  useAccountDefinition,
+  useFiat,
+  useStakedNOTEPoolReady,
+  useStakedNoteData,
+} from '@notional-finance/notionable-hooks';
+import { ExpandedState } from '@tanstack/react-table';
+import { useTheme } from '@mui/material';
+import { Network, lastValue, PORTFOLIO_ACTIONS } from '@notional-finance/util';
+import { Registry, TokenBalance } from '@notional-finance/core-entities';
+import { formatNumberAsPercentWithUndefined } from '@notional-finance/helpers';
+import { useHistory, useLocation } from 'react-router';
+
+export function usePortfolioSNOTETable() {
+  const theme = useTheme();
+  const history = useHistory();
+  const { pathname: currentPath } = useLocation();
+  const [expandedRows, setExpandedRows] = useState<ExpandedState>({});
+  const account = useAccountDefinition(Network.mainnet);
+  const snoteBalance = account?.balances.find((t) => t.symbol == 'sNOTE');
+  const isPoolReady = useStakedNOTEPoolReady();
+  const sNOTEPool = Registry.getExchangeRegistry().getSNOTEPool();
+  const stakedNoteData = useStakedNoteData();
+  const baseCurrency = useFiat();
+  let result: any[] = [];
+  let noStakedNoteData = true;
+
+  const stakeNoteStatus = account?.stakeNOTEStatus;
+
+  if (isPoolReady && snoteBalance && sNOTEPool && stakedNoteData) {
+    const currentSNOTEYield = formatNumberAsPercentWithUndefined(
+      lastValue(stakedNoteData)?.apy,
+      '-',
+      2
+    );
+    const { ethClaim, noteClaim } =
+      sNOTEPool.getCurrentSNOTEClaims(snoteBalance);
+
+    noStakedNoteData =
+      (noteClaim && noteClaim?.isZero()) || noteClaim === undefined
+        ? true
+        : false;
+
+    const currentPrice = TokenBalance.unit(snoteBalance.token)
+      .toFiat('NOTE')
+      .toDisplayStringWithSymbol();
+
+    const sNOTEData = {
+      asset: {
+        symbol: 'sNOTE',
+        symbolSize: 'large',
+        symbolBottom: '',
+        label: 'sNOTE',
+        caption: '80% NOTE 20% ETH',
+      },
+      marketApy: currentSNOTEYield,
+      noteValue: {
+        data: [
+          {
+            displayValue: noteClaim.toDisplayStringWithSymbol(2, true, false),
+            isNegative: false,
+          },
+          {
+            displayValue: noteClaim
+              .toFiat(baseCurrency)
+              .toDisplayStringWithSymbol(2, true, false),
+            isNegative: false,
+          },
+        ],
+      },
+      ethValue: {
+        data: [
+          {
+            displayValue: ethClaim.toDisplayStringWithSymbol(2, true, false),
+            isNegative: false,
+          },
+          {
+            displayValue: ethClaim
+              .toFiat(baseCurrency)
+              .toDisplayStringWithSymbol(2, true, false),
+            isNegative: false,
+          },
+        ],
+      },
+      isDebt: true, // NOTE this is just to make sure the value is note red
```

Comment:
Are you sure this is correct? Why do we want the balance to be red?

Reply:
The sub drawer looks for the '-' to see if a number is negative. I don't think we'd want to show red for any of these values. Also is there even a time when they would be negative?